Source/JavaScriptCore/runtime/ISO8601.cpp

```diff
@@ -843,7 +905,37 @@ static std::optional<CalendarRecord> parseCalendar(StringParsingBuffer<Character
     if (*buffer != ']')
         return std::nullopt;
     buffer.advance();
-    return CalendarRecord { WTFMove(result) };
+
+    return CalendarRecord { isCritical, false, WTFMove(result) };
+}
+
+template<typename CharacterType>
+static std::tuple<std::optional<CalendarRecord>, std::optional<CalendarRecord>>
+parseCalendar(StringParsingBuffer<CharacterType>& buffer)
```

Comment:
Why is it only returning up to two calendars?
From the definition of https://tc39.es/proposal-temporal/#prod-Annotations, it can be a vector of annotations, right?
If so, then it should be `std::optional<Vector<CalendarRecord>>`

Reply:
Done in [9ac33a5](https://github.com/WebKit/WebKit/pull/36361/commits/9ac33a55dfdaf9198aace8a87462d875cf01e0d5)